inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/annotation_fact-extraction.adoc

```diff
@@ -0,0 +1,90 @@
+In projects settings, switch to the *Knowledge Bases* tab, then click *New…* on the bottom and a
+dialogue box shows as in the figure below.
+
+image::factExtraction1.png[align="center"]
+
+To create a local knowledge base supporting qualifiers, one needs to choose *Local* for the type and
+ to choose *WIKIDATA* for the Reification. One can then follow the wizard to finish the setting.
+
+If *NONE* is chosen for the Reification, one cannot add or view qualifiers in the knowledge base.
+
+==== Managing qualifiers in the knowledge base
+Assuming one has already created concepts, properties, instances and a statement about instance
+_Barack Obama_ in this knowledge base:
+
+image::factExtraction2.png[align="center"]
+
+One can click the button *+ Add qualifier*, choose a property _start-date_ for this qualifier name,
+enter _1988_ for this qualifier value, and click *√* to save this qualifier.
+
+image::factExtraction3.png[align="center"]
+
+One can click the pencil icon on the same line as this qualifer to edit and delete this qualifier.
+
+==== Extracting a fact in the annotation page
+Assuming the project contains documents for annotation, one can now switch to the annotation page
+and choose the fact layer:
+
+image::factExtraction4.png[align="center"]
+
+One can now mark the predicate mention of a fact. In the screenshot below, _enrolled in_ is selected
+. The right sidebar shows feature editors for the predicate, the subject, the object and the
+qualifiers. One can then choose _educated-at_ in the dropdown field of the predicate feature editor
+to link the mention of this predicate to the property in the knowledge base. The candidate list of
+this dropdown field is a list of properties in the knowledge base.
+
+image::factExtraction5.png[align="center"]
+
+To annotate the subject mention of a fact, one needs to click *<Click to activate>* in the subject
+feature editor. After this input field turns orange, one can now mark the subject mention _Obama_of
+a fact. One can then choose _Barack Obama_ in the dropdown field of the subject feature editor to
+link the mention of this subject to the instance in the knowledge base. The candidate list of this
+dropdown field is a list of instances in the knowledge base. In the main editor, there is an arrow
+from the label above the predicate to the label above the subject, with a name _subject_.
+
+image::factExtraction6.png[align="center"]
+
+Extracting the object mention is same as extracting the subject.
+
+If this fact is already saved in the knowledge base, the label in the predicate feature editor shows
+ *There is at least one statement in the KB which matches for this SPO*.
+
+To extract a qualifier, one needs to choose the qualifier name from the drop-down field in front of
+the button *add* in the qualifier feature editor. The list of candidates for the qualifier name is a
+ list of properties from the knowledge base. After clicking *add*, a mention input field and a
```

Comment:
It doesn't say here form which KB (if there is more than one) the properties are obtained.